packages/core/src/DotLoader/DotLoader.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, { css, keyframes } from 'styled-components'
+import { Box } from '../Box'
+import { Flex } from '../Flex'
+import { applySizes } from '../utils'
+
+const sizes = {
+  small: css`
+    height: 18px;
+
+    & > * {
+      width: 8px;
+      height: 8px;
+    }
+
+    & > :not(:last-child) {
+      margin-right: 16px;
+    }
+  `,
+  medium: css`
+    height: 21px;
+
+    & > * {
+      width: 12px;
+      height: 12px;
+    }
+
+    & > :not(:last-child) {
+      margin-right: 32px;
+    }
+  `,
+  large: css`
+    height: 24px;
+
+    & > * {
+      width: 16px;
+      height: 16px;
+    }
+
+    & > :not(:last-child) {
+      margin-right: 32px;
```

Comment:
A good chunk of these hardcoded pixel values could be replaced with `themeGet()`s 👀

Reply:
not anymore they're not! good catch 👍